src/types/Comment.ts

```diff
@@ -0,0 +1,7 @@
+export type Comment = {
```

Comment:
Consider using an interface instead of a type alias for defining the Comment object, as interfaces provide more extensibility.